pom.xml

```diff
@@ -69,20 +69,6 @@
 
     <dependencyManagement>
         <dependencies>
-            <!--Child modules-->
-            <dependency>
-                <groupId>org.bahmni.module</groupId>
```

Comment:
@sivareddytw why the api and omod dependency removed from main pom.xml file ?

Reply:
@binduak bahmni-commons-omod is removed as it not used as a dependency in any of modules. 
bahmni-commons-api is retained